### DISCUSSION TITLE
第5节课笔记(23班)

### DISCUSSION THREAD

**Discussion by wangyijunlyy** [InternLM/Tutorial#419] [笔记]
笔记提交

**Comment by i-square** [comment]
https://i-square.github.io/p/InternLM-tutorial-campsection5-LLM-Quantization-Deployment-Practice-based-on-LMDeploy/

**Comment by PlateAss** [comment]
https://plateass.github.io/class5

**Comment by YangZhC** [comment]
https://blog.csdn.net/donotpython/article/details/135765347

**Comment by Epi10gue** [comment]
http://t.csdnimg.cn/0xcEe
